test/test.js

```diff
@@ -1,5 +1,52 @@
+const path = require("path")
 const test = require("tape").test
+const FlyNotify = require("../")
+const defaults = {
+  title: "Fly",
+  message: "Task completed.",
+  icon: path.resolve(__dirname, "..", "fly-logo.png"),
+}
+const plugin = new FlyNotify()
 
-test("fly-notify", function (t) {
- t.end()
+// Adding this to make sure tests pass since we're not checking that the `node-
+// notifier` functionality works.
+plugin.error = function() {}
+plugin.debug = function() {}
+
+test("Testing `fly-notify`", function (t) {
+  t.plan(2)
+  t.ok(plugin.notify, "Fly-Notify `notify` method exists ~~%")
+  t.equal(typeof plugin.notify, "function", "Fly-Notify `notify` method is a function ~~%")
+  t.end()
+})
+
+test("Testing bad options", function(t) {
```

Comment:
Notice that

```js
const babel = require(bind("../"))
```

You only need to do that if your plugin is written in ES6/7.

Also notice we are "hijacking" `babel` with our own object that has a mock of `fly` with `filter`.